### PR TITLE
Wip configuration plan

### PR DESCRIPTION
**Pull request by pchelle**
@abdullahhamadeh the PR is a draft of how I would start including the configuration plan into a qualification workflow
The features allows to create a first report with all the content, intro and inputs
They also provide folders to store the simulation and pk analysis results
One issue is that I used the simulationSet objects to do so, but did not fill its output paths (so simulation results are currently empty) 

Let me know if you have questions

### CODE REVIEW

**Comment by msevestre** [comment]
@pchelle Can you update base on my review if possible today
Then @abdullahhamadeh can you rebase/merge your other PR on develop so that we can split the changes? I feel that your PR #481 was started based off #479  and this will be hard to follow

@pchelle Can you also review the PR #481 ?

File: R/configuration-plan.R [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+      }
+      addTextChunk(fileName = self$getSectionMarkdown(input$SectionId), text = readLines(inputLocation), logFolder = logFolder)
+      return(invisible())
+    },
+    
+    #' @description Copy intro to intro markdown
+    #' @param intro list including Path
+    #' @param logFolder path where logs are saved
+    copyIntro = function(intro, logFolder = getwd()) {
+      validateIsIncluded(names(intro), "Path")
```
[the remark this answers is not in the document]
Comment (pchelle) [reply R/configuration-plan.R:101]: This is partly done here: 
- the field "Path" defined within "Intro" of the configuration plan (json file) is mandatory as it is where the content of intro is read
- The expected type of intro is a "list" as it is read from `jsonlite::fromJSON`, but it should include the mandatory field "Path". Any R object with such a field is actually also okay (e.g. R6 object or a data.frame). So, in case someone needs to tune the `intro` field of the configuration plan, I did not check the exact type of `intro`.

Let me know if that is suitable

**Review by msevestre — APPROVED** [review]
